libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -442,7 +443,7 @@ export const transformTemplateToMobile = (popup, item, localnav = false) => {
       </span>
     </div>
     <div class="title">
-      <div class="breadcrumbs"></div>
+      ${breadCrumbs}
```

Comment:
do we want to do `${isHomePage() : '' : breadcrumbs}`?

Reply:
We are not showing breadcrumbs in homepage

Reply:
Yes but we need that empty breadcrumbs div to make sure we have the right amount of space above the mega menu title.